lib/lambda-worker/lambda-worker-props.ts

```diff
@@ -0,0 +1,51 @@
+import * as cdk from "@aws-cdk/core";
+import * as ec2 from "@aws-cdk/aws-ec2";
+import * as iam from "@aws-cdk/aws-iam";
+import * as sns from "@aws-cdk/aws-sns";
+
+export interface LambdaWorkerProps {
+  // The name of the LambdaWorker
+  name: string;
+
+  // Lambda Properties
+  // Documented here https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-lambda-nodejs.NodejsFunctionProps.html
+  lambdaProps: {
```

Comment:
Does it cover all of them? If so, could you use `lambdaProps: NodejsFunctionProps ` instead?